app/routes.py

```diff
@@ -1,2 +1,89 @@
-from flask import Blueprint
+from flask import Blueprint , jsonify, request , make_response
+from app import db
+from app.models.task import Task 
 
+tasks_bp = Blueprint("tasks", __name__ , url_prefix="/tasks")
+
+@tasks_bp.route("", methods =["POST","GET"])
+def manage_tasks():
+    if request.method == "POST":
+        request_body = request.get_json()
+        try:
+            new_task = Task(
+                title = request_body["title"],
+                description= request_body["description"],
+                completed_at = request_body["completed_at"]
+            )
+            #add new task before response body to get an id value
+            db.session.add(new_task)
+            db.session.commit()
+            response_body = {
+                "task":{
+                "id": new_task.task_id,
+                "title": new_task.title,
+                "description": new_task.description,
+                "is_complete": True if new_task.completed_at is not None else False
+                }
+            }
+            return jsonify(response_body), 201
+        except KeyError:
+            return {"details":"Invalid data"}, 400
+
+    elif request.method == "GET":
+        request_title = request.args.get("title")
```

Comment:
to implement sorting you can do something similar to this: 
`sort_method = request.args.get('sort')`
then create a conditional that says if the sort method equals "asc" then orcer_by asc and the same for descending